src/inputs/select/select.jsx

```diff
@@ -192,6 +196,7 @@ const defaultProps = {
     noResultsText: 'No results found',
     onChange: null,
     onClose: undefined,
+    onInputChange: null,
```

Comment:
NAB: `undefined` is probably better here than `null`, not that it matters much.
I guess I like `null` as a default for _data_ (like this prop can be some specific object/type, or `null`), but for things like optional CSS classes or styles or event handler functions, I think `undefined` is appropriate.